lib/matplotlib/colors.py

```diff
@@ -714,20 +714,26 @@ class Colormap:
     chain.
     """
 
-    def __init__(self, name, N=256):
+    def __init__(self, name, N=256, *, bad=None, under=None, over=None):
         """
         Parameters
         ----------
         name : str
             The name of the colormap.
         N : int
             The number of RGB quantization levels.
+        bad : :mpltype:`color`, default: transparent
+            The color for invalid values (NaN or masked).
```

Comment:
```suggestion
            The color for invalid values (NaN or masked).
            .. versionadded:: 3.11
```
? (Likewise in many locations below. Hence, maybe it is not required...)

Reply:
Done. It's a bit verbose, but OTOH it's good to be explicit about this.